client/upgrade.lua

```diff
@@ -72,7 +72,6 @@ local function loadWindowsFFI()
     typedef const CHAR *LPCSTR, *PCSTR;
 
     enum {
-    ERROR_SUCCESS = 0L,
```

Comment:
It's defined by the libc headers usually. I was getting a warning of it being doubly defined.